lib/setup/setupSetup.js

```diff
@@ -217,42 +220,56 @@ function Setup(options) {
                     op = 9001;
                 } else if (otype === 'redis') {
                     op = 6379;
+                    if (ohost.includes(',')) {
+                        ohost = ohost.split(',');
+                        ohost.forEach((host, idx) => {
+                            ohost[idx] = host.trim();
+                        });
+                        op = 26379;
+                    }
                 }
 
                 rl.question('Port of objects DB(' + otype + '), default[' + op + ']: ', userPort => {
-                    let ot;
                     let oport;
                     if (!userPort) {
-                        if (otype === 'file') {
-                            oport = 9001;
-                            ot = 'file';
-                        } else if (otype === 'redis') {
-                            ot = 'redis';
-                            oport = 6379;
-                        }
+                        oport = op;
                     } else {
-                        oport = parseInt(userPort, 10);
-                        if (isNaN(oport)) {
-                            console.log('Invalid objects port: ' + oport);
-                            callback(23);
+                        if (userPort.includes(',')) {
+                            oport = userPort.split(',');
+                            oport.forEach((port, idx) => {
+                                oport[idx] = parseInt(port.trim(), 10);
+                                if (isNaN(oport[idx])) {
+                                    console.log('Invalid objects port: ' + oport[idx]);
+                                    callback(23);
```

Comment:
I think this construct (forEach plus callback) can result in the callback method being called multiple times. For process.exit that doesn't matter, but it is possible that for some application callback is NOT `process.exit`.

I believe this should be a classic for loop with a `break` statement (or even `return`) after `callback(23)`.

Reply:
I added it, but in fact in standard case "callback " will be process exit :-)

Reply:
Ok lets deal with it when we get a bug report ;)

Reply:
:-)